lib/ansible/module_utils/aws/elbv2.py

```diff
@@ -717,7 +717,7 @@ def _compare_condition(self, current_conditions, condition):
                         current_condition['SourceIpConfig']['Values'][0] == condition['SourceIpConfig']['Values'][0]):
                     condition_found = True
                     break
-            elif current_condition['Field'] == condition['Field'] and current_condition['Values'][0] == condition['Values'][0]:
+            elif current_condition['Field'] == condition['Field'] and current_condition['Values'] == condition['Values']:
```

Comment:
Should the ['Values'] lists be sorted before the comparison?

Reply:
Good point, most of the time I anticipate `['Values']` will only be one item. But in #65021 it does support multiple values, which are a list of hostnames. I don't know if AWS returns that list in the same order all the time. Nevertheless, if someone made changes in the AWS console and then applied those changes through ansible, it is possible that the hostname order would be different and Ansible would report the configuration changed.

Reply:
I have submitted #65315 for devel which should address this there. I'm not sure how to submit a change to someone elses PR.

Reply:
@roberbri changes need to be merged to `devel` before they can be backported, so you can't add something directly to this PR without going via `devel` first. Assuming #65315 gets merged before this PR gets merged, I can include a backport of it in this PR. If not, it can be added later with another backport PR.